datagenerator/src/main/java/com/linkedin/restli/datagenerator/csharp/CSharpMap.java

```diff
@@ -34,11 +34,11 @@ public CSharpMap(ClassTemplateSpec spec, CSharpDataTemplateGenerator dataTemplat
   }
 
   /**
-   * If param modifier is MUTABLE:
+   * If param modifier is MUTABLE_SHALLOW:
    *  Returns a mutable map of immutable types. The reason for this is that
    *  C# cannot handle casting a nested list/dictionary object; it can only handle
    *  casting the outer layer. Thus, the inner types must be immutable.
-   * If param modifier is DATAMAP_PARSE:
+   * If param modifier is TYPED_DATAMAP:
    *  Returns a mutable map of types that are readable by the constructors
    *  of each datamodel. For instance, all record types would be represented
    *  as Dictionary(string, object)
```

Comment:
Dictionary<string, object>. Same as previous comment, this seems contra-dictionary to TYPED_DATAMAP? Similar to my previous comment, maybe we can remove these comments since we have detailed comment in NameModifier enum.